src/components/access-type.js

```diff
@@ -542,6 +548,10 @@ const mapDispatchToProps = dispatch => ({
  * ```javascript
  * import { AccessType } from "@quintype/components";
  *
+ * propName | Description
+ *  --- | ---
+ *  prodHost | It is passed to configure the production host.(Optional)
+ *  stagingHost | It is passed to configure the staging host.(Optional)
```

Comment:
The format does not seem to align with the existing docs right ?
@ags1773

Reply:
@cricket-lover pls move this block of code on top, right after `AccessType is a generic connected render prop which exposes methods to handle access to stories / assets and initialize accesstype js`